<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compressor2</class>
 <widget class="QMainWindow" name="compressor2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>compressor2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Show_pic">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Show_pic_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>21</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../projection-20.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Another compression operation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="picture_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="placeholderText">
     <string>picture path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse_enc">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse enc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decompress">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compress">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compression Ratio</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="frq_path">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>frq path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="comp_ratio">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="enc_path">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>enc path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Browse_frq">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse frq</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
